sigopt/interface.py

```diff
@@ -161,6 +160,7 @@ def __init__(self, requestor, api_url=None, user_agent=None, verify_ssl_certs=No
         suggestions,
         experiment_tokens,
         experiment_training_runs,
+        best_training_runs,
```

Comment:
This got doubled-up (it's also second in this list)

Reply:
fixed! protip in vim: visual select the list content, then `!sort | uniq` to replace with sorted+unique version

Reply:
ohh, very cool!